### PR TITLE
Feature/risk endpoint

### PR DESCRIPTION
**Pull request by yorickdewid**
**What issue does this PR address?**
Request risk from the portal endpoint

**Does this PR introduce a breaking change?**
Nope

### CODE REVIEW

File: src/FunderMaps.WebApi/DataTransferObjects/AnalysisRiskDto.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    {
+        /// <summary>
+        ///     Internal neighborhood id in which this building lies.
+        /// </summary>
+        public string NeighborhoodId { get; set; }
+
+        /// <summary>
+        ///     Represents the foundation type of this building.
+        /// </summary>
+        public FoundationType FoundationType { get; set; }
```
Comment (tabeckers) [inline src/FunderMaps.WebApi/DataTransferObjects/AnalysisRiskDto.cs:19]: Enums should be nullable as well

File: src/FunderMaps.Core/Services/ProductService.cs [hunk 2]
```diff
@@ -147,18 +154,15 @@ public virtual Task<IEnumerable<AnalysisProduct>> GetAnalysisInFenceAsync(Guid u
         /// <param name="userId">Internal user id.</param>
         /// <param name="productType"><see cref="StatisticsProductType"/></param>
         /// <param name="neighborhoodCode">Neighborhood code.</param>
-        /// <param name="navigation"><see cref="INavigation"/></param>
         /// <param name="token"><see cref="CancellationToken"/></param>
         /// <returns><see cref="StatisticsProduct"/></returns>
-        public virtual async Task<StatisticsProduct> GetStatisticsByNeighborhoodAsync(Guid userId, StatisticsProductType productType, string neighborhoodCode, INavigation navigation, CancellationToken token = default)
+        public virtual async Task<StatisticsProduct> GetStatisticsByNeighborhoodAsync(
+            Guid userId,
```
Comment (tabeckers) [inline src/FunderMaps.Core/Services/ProductService.cs:160]: Statistic products should also be checked against user id. I suggest making an issue for this matter and leaving it as is for now, agreed?

**Review by tabeckers — CHANGES_REQUESTED** [review]
Left some comments, open for discussion

File: src/FunderMaps.Core/Services/ProductService.cs (hunk 2 above)
Comment (yorickdewid) [reply src/FunderMaps.Core/Services/ProductService.cs:160]: I'll add thi

**Review by tabeckers — APPROVED** [review]

**Pull request merged**